e2e/components/Slider/Slider-test.avt.e2e.js

```diff
@@ -92,6 +92,45 @@ test.describe('@avt Slider', () => {
     await expect(page).toHaveNoACViolations('Slider-with-layer');
   });
 
+  test('@avt-advanced-states slider with custom format', async ({ page }) => {
+    await visitStory(page, {
+      component: 'Slider',
+      id: 'components-slider--slider-with-custom-value-label',
+      globals: {
+        theme: 'white',
+      },
+    });
+
+    await expect(page).toHaveNoACViolations('Slider-with-custom-value-label');
+  });
+
+  test.slow('@avt-keyboard-nav slider with custom format', async ({ page }) => {
```

Comment:
It is missing the visitStory here
```suggestion
  test.slow('@avt-keyboard-nav slider with custom format', async ({ page }) => {
  await visitStory(page, {
      component: 'Slider',
      id: 'components-slider--slider-with-custom-value-label',
      globals: {
        theme: 'white',
      },
    });
```

Reply:
Should be fixed now